examples/scripts/SPMe_SOC.py

```diff
@@ -85,15 +85,16 @@
         xnsurf = sol["X-averaged negative particle surface concentration"]
         time = sol["Time [h]"]
         # Coulomb counting
-        time_hours = time(sol.t)
+        time_secs = sol.t * model.timescale_eval
```

Comment:
maybe do `sol["Time [s]"].entries` to avoid having to call `model.timescale_eval`?

Reply:
agreed that  the recommended way should be `sol["Time [s]"].entries`. I've made this consistent everywhere (I think)